src/styles/base/_override.typography.scss

```diff
@@ -15,123 +15,113 @@
 .ds-text-heading--lg,
 .ds-text-heading--md,
 .ds-text-heading--sm {
+  font-family: Montserrat;
```

Comment:
```suggestion
  font-family: $montserrat-font;
```

Reply:
This was fixed in the other branch and was worried this might cause merge issues if we fix them in both. 
https://github.com/CMSgov/mgov-design-system/pull/83

Reply:
Ah gotcha, if you merge the other branch them you wouldn't need to include the update here as well. However updating  it here as well wouldn't cause any issues

Reply:
Cool! That's good to know, I merged the other branch.